### PR TITLE
add random currencies

### PR DESCRIPTION
**Pull request by ezeql**
add random Currencies() from ISO 4217

### CODE REVIEW

File: random_data_test.go [hunk 1]
```diff
@@ -133,6 +133,13 @@ func TestCountry(t *testing.T) {
 	}
 }
 
+func TestCurrency(t *testing.T) {
+	t.Log("TestCurrency")
+	if !findInSlice(jsonData.Currencies, Currency()) {
+		t.Error("Couldnt find currency in urrencies")
```
Comment (Pallinder) [inline random_data_test.go]: Typo here.

**Comment by Pallinder** [comment]
Great!

**Comment by Pallinder** [comment]
Should probably add an example to the readme as well.

**Comment by ezeql** [comment]
ops! forgot to add an example on the readme. I'll add it asap.

**Comment by Pallinder** [comment]
Awesome. Merging.

**Pull request merged**